src/services/jwt.service.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY, { expiresIn: '30m' });
```

Comment:
Consider adding more claims to the JWT payload, such as 'iat' (issued at) or 'sub' (subject), to enhance security and traceability.